silverback/exceptions.py

```diff
@@ -35,7 +35,7 @@ def __init__(self):
 
 
 class CircuitBreaker(SilverbackException):
```

Comment:
It causes mental obstacles when exceptions don't end in the word `Error`. When I see `raise InvalidContainerType` -- like, "why am I raising a type?".

This is unconventional!

Reply:
This has a special intent to be driven by user logic-specific scenarios, so I used a common word to the financial sphere

Reply:
meh

Reply:
sorry I commented on CircuitBreaker, but the ones I really dont like are:

```
InvalidContainerType
DuplicateHandler
```

Reply:
Can fix those for sure

Reply:
Fixed in 3d49227